src/main/scala/com/gd/twitterstreamingtToDatalake/StreamingTweetsJob.scala

```diff
@@ -0,0 +1,38 @@
+package com.gd.twitterstreamingtToDatalake
+
+import org.apache.log4j.{Level, Logger}
+
+
+object StreamingTweetsJob extends SparkSessionWrapper {
+
+  //Set the Logger Level
+
+  Logger.getLogger("org").setLevel(Level.ERROR)
+
+  def main(args: Array[String]): Unit = {
+
+    val englishTweets = tweetsIngestion.getTweets
+
+    val hashTags = transformTweets.getText(englishTweets)
+
+
+
+
+
+
+
```

Comment:
I believe the empty lines are redundant.
Please keep attention to formatting.
The goal is to keep codebase clean and beautiful :)

Reply:
Fixed..